packages/core/src/types.ts

```diff
@@ -21,7 +24,7 @@ export interface InitOptions {
   /**
    * The chains that your app works with
    */
-  chains: Chain[]
+  chains: Chain[] | ChainWithDecimalId[]
```

Comment:
if change the types to `type ChainId = number | string` then we could have just `Chain[]` vs `Chain[] | ChainWithDecimalId[]`